classes/actions/Coup.py

```diff
@@ -1,11 +1,8 @@
 from .Action import Action
 
 class Coup(Action):
-    def __init__(self, target : int, card_id : int = None):
+    def __init__(self, target : int = None, card_id : int = None):
```

Comment:
why is default None?

Reply:
in new commit, it allows not having to pass a target when checking if the player can coup. But I think this will go away if I do the above change